projects/Mallard/src/helpers/push-download-failsafe.ts

```diff
@@ -17,7 +17,7 @@ const pushDownloadFailsafe = (client: ApolloClient<object>) => {
 			await pushTracking('backgroundFetch', 'started', feature);
 			await prepareAndDownloadTodaysIssue(client);
 			await pushTracking('backgroundFetch', 'ended', feature);
-			BackgroundFetch.finish(BackgroundFetch.FETCH_RESULT_NEW_DATA);
+			BackgroundFetch.finish();
```

Comment:
Does this makes in any difference in terms of download?

Reply:
Shouldnt do from what I can see.